docs/reference/method/MongoDBGridFSBucket-registerGlobalStreamWrapperAlias.txt

```diff
@@ -0,0 +1,132 @@
+===========================================================
+MongoDB\\GridFS\\Bucket::registerGlobalStreamWrapperAlias()
+===========================================================
+
+.. versionadded:: 1.18
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\GridFS\\Bucket::registerGlobalStreamWrapperAlias()
+
+   Registers an alias for the bucket, which enables files within the bucket to
+   be accessed using a basic filename string (e.g.
+   `gridfs://<bucket-alias>/<filename>`).
+
+   .. code-block:: php
+
+      function registerGlobalStreamWrapperAlias(string $alias): void
+
+Parameters
+----------
+
+``$alias`` : array
+  A non-empty string used to identify the GridFS bucket when accessing files
+  using the ``gridfs://`` stream wrapper.
+
+Behavior
+--------
+
+After registering an alias for the bucket, the most recent revision of a file
+can be accessed using a filename string in the form ``gridfs://<bucket-alias>/<filename>``.
+
+Supported stream functions:
+
+- ``copy()``
```

Comment:
Sorry to be a bother, but since these are PHP core functions, did you want to link to their docs pages? OK it not, just thought I'd ask in case you weren't aware that we had a shortcut for that.

For instance, this would be:

```
:php:`copy() <copy>`
```

The text in the angle brackets gets appended to `http://php.net/` (see: https://github.com/mongodb/docs-php-library/blob/master/conf.py#L60).

Edit: I just realized the "php" link doesn't use HTTPS. Opened https://github.com/mongodb/docs-php-library/pull/57 to correct that.

Reply:
Since it's that simple, I've added the link